notebooks/01_prepare_data/wikidata.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>here we could have a filename that depends on the dataset size. This could be defined in the parameters cell</p>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/778/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Implemented here: 9c03398